<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogoSpeech Studio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/icons/app_icon.png</normaloff>:/resources/icons/app_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QCustomPlot" name="chartWAVFile" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QSplitter::handle {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
stop:0 rgba(255, 255, 255, 0), 
stop:0.407273 rgba(200, 200, 200, 255), 
stop:0.4825 rgba(101, 104, 113, 235), 
stop:0.6 rgba(255, 255, 255, 0));
    image: url(:/images/splitter.png);
     }</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QCustomPlot" name="chartEnergy" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QCustomPlot" name="chartPitch" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QCustomPlot" name="chartSpectrogram" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="infoPanel" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="FolderWidget" name="folderWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionSpanish"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="actionRecordSettings"/>
    <addaction name="actionProcessSettings"/>
    <addaction name="actionChartSettings"/>
    <addaction name="separator"/>
    <addaction name="actionEstimate_Levels"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowTerminal"/>
    <addaction name="separator"/>
    <addaction name="actionShowEnergy"/>
    <addaction name="actionShowPitch"/>
    <addaction name="actionShowSpectrogram"/>
    <addaction name="separator"/>
    <addaction name="actionFull_Screen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionFollow_Us"/>
    <addaction name="actionLike_This"/>
    <addaction name="actionSource_Code"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_TFG"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Settings Toolbar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPreview"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Record Toolbar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecord"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
  </widget>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/record.png</normaloff>:/resources/icons/record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="toolTip">
    <string>Click to record speech signals </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/open.png</normaloff>:/resources/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing audio file </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/save.png</normaloff>:/resources/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current signal </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/print.png</normaloff>:/resources/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print the current signal processing </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/exit.png</normaloff>:/resources/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/full_screen.png</normaloff>:/resources/icons/full_screen.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Toggle Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionRecordSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/record_tools.png</normaloff>:/resources/icons/record_tools.png</iconset>
   </property>
   <property name="text">
    <string>Record Settings</string>
   </property>
  </action>
  <action name="actionProcessSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/process_tools.png</normaloff>:/resources/icons/process_tools.png</iconset>
   </property>
   <property name="text">
    <string>Processing Settings</string>
   </property>
  </action>
  <action name="actionChartSettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/chart_tools.png</normaloff>:/resources/icons/chart_tools.png</iconset>
   </property>
   <property name="text">
    <string>Chart Settings</string>
   </property>
  </action>
  <action name="actionAbout_TFG">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/app_icon.png</normaloff>:/resources/icons/app_icon.png</iconset>
   </property>
   <property name="text">
    <string>About TFG</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/qt.png</normaloff>:/resources/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionShowEnergy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Energy  Chart</string>
   </property>
  </action>
  <action name="actionShowPitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pitch Chart</string>
   </property>
  </action>
  <action name="actionShowSpectrogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectrogram</string>
   </property>
  </action>
  <action name="actionShowTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/play.png</normaloff>:/resources/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/overview.png</normaloff>:/resources/icons/overview.png</iconset>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </action>
  <action name="actionFollow_Us">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/twitter.png</normaloff>:/resources/icons/twitter.png</iconset>
   </property>
   <property name="text">
    <string>Follow Us</string>
   </property>
  </action>
  <action name="actionLike_This">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/facebook.png</normaloff>:/resources/icons/facebook.png</iconset>
   </property>
   <property name="text">
    <string>Like This</string>
   </property>
  </action>
  <action name="actionSource_Code">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/github.png</normaloff>:/resources/icons/github.png</iconset>
   </property>
   <property name="text">
    <string>Source Code</string>
   </property>
  </action>
  <action name="actionEstimate_Levels">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/resources/icons/levels.svg</normaloff>:/resources/icons/levels.svg</iconset>
   </property>
   <property name="text">
    <string>Estimate Levels</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FolderWidget</class>
   <extends>QWidget</extends>
   <header location="global">customwidget/folderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
